packages/forms/src/directives/validators.ts

```diff
@@ -159,21 +175,37 @@ export class RequiredValidator implements Validator {
     if (this._onChange) this._onChange();
   }
 
+  /**
+   * @description
+   * Method that receives a control to validate against if required.
```

Comment:
Might be easier to understand this way (had to read it a few times because I thought it was saying we were validating whether or not the control was required, rather than only validating if the control was required)

```suggestion
   * Method that validates whether the control is empty.
```